app/models/user.rb

```diff
@@ -0,0 +1,4 @@
+class User < ActiveRecord::Base
+  has_many :decks
+  has_many :cards, through: :decks
```

Comment:
when using has_many through I like to explicitly state if the collection should be unique or allow duplicates